frontend/public/components/utils/dropdown.jsx

```diff
@@ -192,6 +192,7 @@ class DropDownRowWithTranslation extends React.PureComponent {
         <a
           href="#"
           ref={this.link}
+          data-test={`${itemKey}-link`}
```

Comment:
There is already one `data-test` attribute
```
          data-test="dropdown-menu-item-link"
```
  197:11  error  No duplicate props allowed  react/jsx-no-duplicate-props

Reply:
This is a bit tricky one (for me). @dtaylor113 has added this line/change with this PR: https://github.com/openshift/console/pull/7714 . @dtaylor113 , can you please take a look and see whether we can make this more generic as suggested in this PR.

Reply:
In this file `packages/integration-tests-cypress/views/common.ts` we are using this id. So here we have to make sure we use the generic name for `cy.byTestID()` method.

Reply:
Instead of using:
`cy.byTestID(`${scName}-link`).click();`
can you use:
`cy.byTestID('dropdown-menu-item-link').contains(`${scName}`).click();`

Reply:
@dtaylor113 , tried the above sample code, but it didn't work. )=

Reply:
@dtaylor113 , sorry my bad... Somehow the new code change was not compiled and the tests were failing.
I ran `yarn run dev` once and then ran the cypress tests, the change worked... Thanks